src/aro/Driver.zig

```diff
@@ -310,20 +309,15 @@ pub fn parseArgs(
             } else if (mem.eql(u8, arg, "-fnative-half-arguments-and-returns")) {
                 d.comp.langopts.allow_half_args_and_returns = true;
             } else if (mem.eql(u8, arg, "-fno-pic") or mem.eql(u8, arg, "-fno-PIC") or mem.eql(u8, arg, "-fno-pie") or mem.eql(u8, arg, "-fno-PIE")) {
```

Comment:
We can just remove these `if` statements entirely

Reply:
Without these if statements, if there are options related to pic/pie in the arguments, it will directly report an error of type [`.cli_unknown_arg`.](https://github.com/Vexu/arocc/blob/406f323eb97aa2dd2e339ba194e28bfdc3a689cb/src/aro/Driver.zig#L509)